acoustic_msgs/msg/RawSonarImage.msg

```diff
@@ -1,35 +1,39 @@
-
-# The header time should be the time the ping was transmitted, not
-# the time the message was received.
-# Note that these two could potentially be several seconds apart,
-# especially in deep water.
+# All values in the header are associated the transmitter:
+# 	- stamp references the transmit time or the start of the
+#	  transmit cycle.
+#	- frame_id references the transmitter's acoustic center
 std_msgs/Header header
 
 PingInfo ping_info
 
-float32 sample_rate   # hz
-
 uint32  samples_per_beam  # the number of samples in each beam
 
 # Many sonars have some kind of upper gate for water column data
 # this represents the sample number of the first non empty sample
 # for beam n
 uint32 sample0
 
+# if the image is not reported at the full resolution
+# ping_info.sample_rate != image_sample_rate
+# for a given sample:
+#   2-way travel time = image_sample_rate*(sample_no+sample0)
```

Comment:
This equation doesn't match how I expect "rate" to be used -- here, image_sample_rate has units of seconds, so I'd have expected it to be called "sampling period".

can we change this to be `(sample_number + sample0) / image_sample_rate`?